WebApp/Controllers/ContainerController.cs

```diff
@@ -8,19 +8,12 @@ namespace WebApp.Controllers
     [ApiController]
     public class ContainerController : ControllerBase
     {
-        private readonly IContainer<int> _container;
-
-        public ContainerController(IContainer<int> container)
-        {
-            _container = container;
-        }
-
-        [HttpGet]
-        public IActionResult Get()
+        private readonly Container<int> _container;
+        public ContainerController(Container<int> containerController)
```

Comment:
To jest container, a nie containerController.  To chyba pozostałość ze starego PR